src/CMakeLists.txt

```diff
@@ -217,6 +217,24 @@ endif(ENABLE_REDIS)
 
 find_package(SQLite3 REQUIRED)
 
+OPTION(ENABLE_PROMETHEUS "Enable prometheus client support" FALSE)
```

Comment:
+`set(USE_PROMETHEUS FALSE)`

Reply:
exact, missed that one

Reply:
(note: compilers seems to be compliant if not defined too :D)